migrations/initdb-triggers.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}triggers`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.createTable(table, {
+                id: {
+                    allowNull: false,
+                    autoIncrement: true,
+                    primaryKey: true,
+                    type: Sequelize.INTEGER
+                },
+                src: {
+                    type: Sequelize.STRING(64)
+                },
+                dest: {
+                    type: Sequelize.STRING(64)
+                }
+            }, { transaction }
+            );
+
+            await queryInterface.addConstraint(table, ['src', 'dest'],
+                {
+                    name: `${table}_src_dest_key`,
+                    type: 'unique',
+                    transaction
+                }
+            );
+
+            await queryInterface.addIndex(table, ['dest', 'src'],
+                {
+                    name: `${table}_dest`,
+                    transaction
+                }
```

Comment:
According to [this schema definition](https://github.com/screwdriver-cd/data-schema/blob/master/models/trigger.js#L59), this index should be separated into `dest` and `src`, not composite index.

Reply:
This index was changed per access pattern